city_scrapers/spiders/fortx_Tarrant_County_Commissioners_Court.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+import scrapy
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse as dateparse
+
+
+class FortxTarrantCountyCommissionersCourtSpider(CityScrapersSpider):
+    name = "fortx_Tarrant_County_Commissioners_Court"
+    agency = "Tarrant County Commissioners Court"
+    timezone = "America/Chicago"
+
+    """
+    The scraper gathers meeting information from two separate endpoints
+    specified in start_urls. One endpoint provides archived meetings,
+    and the other provides current and upcoming meetings. This approach
+    was chosen to keep the scraper simple and avoid overcomplicating it,
+    especially since the CSS of the source webpages was quite messy.
+    """
+    start_urls = [
+        "https://tarrant-agendamanagement-public.techsharetx.gov/publicportal/api/meetings/readArchived",  # noqa
+        "https://tarrant-agendamanagement-public.techsharetx.gov/publicportal/api/meetings/readCurrentAndUpcoming",  # noqa
+    ]
+    agenda_base_url = "https://prod-agendamanagement-publicportal.azurewebsites.us/HtmlAgenda/"  # noqa
+    committee_id = "fe6aa5cc-7448-4194-ac6e-08dc95f79ccc"
+
+    source_url = "https://www.tarrantcountytx.gov/en/commissioners-court/commissioners-court-agenda-videos.html"  # noqa
+
+    location = {
+        "address": "100 East Weatherford Street, 5th Floor, Fort Worth, Texas 76196",  # noqa
+        "name": "Tarrant County Administration Building (check the agenda for room location)",  # noqa
+    }
+
+    def start_requests(self):
+        payload = {"committeeId": self.committee_id}
+        for url in self.start_urls:
+            yield scrapy.Request(
+                url=url,
+                method="POST",
+                body=json.dumps(payload),
+                headers={"Content-Type": "application/json"},
+                callback=self.parse,
+            )
+
+    def parse(self, response):
+        data = json.loads(response.text)
```

Comment:
i think scrapy already comes with the ability to convert the response to json. try

```suggestion
        data = response.json()
```

see [L25 on my PR](https://github.com/cruznunez/city-scrapers-losca/blob/spider-losca_City_Council/city_scrapers/spiders/losca_City_Council.py#L25)